.travis.yml

```diff
@@ -7,6 +7,8 @@ branches:
     - /^\d\.\d+\.\d+(rc\d+|\.dev\d+)?$/
 matrix:
   include:
+    - env: TOXENV=flake8
+      python: 3.5
```

Comment:
Shouldn’t we use 3.7? (also relevant for `tox.ini`)

Reply:
It's ok for me, in fact I don't remember why I put 3.5.... Already pushed a commit with this change, thank you :+1: